src/com/google/javascript/jscomp/VarCheck.java

```diff
@@ -303,7 +303,12 @@ public void visit(NodeTraversal t, Node n, Node parent) {
             }
             // fall through
           default:
-            t.report(n, NAME_REFERENCE_IN_EXTERNS_ERROR, n.getString());
+            // Don't warn for simple var assignments "/** @const */ var foo = bar;"
+            // They are used to infer the types of namespace aliases.
+            if (parent.getType() != Token.NAME || parent.getParent() == null ||
+                parent.getParent().getType() != Token.VAR) {
```

Comment:
May as well allow LET and CONST too as we'll probably want to allow ES6 in externs at some point (though we don't yet)